packages/theme/pages/MyAccount/AddressesDetails.vue

```diff
@@ -135,7 +145,7 @@ export default defineComponent({
       const isDefault = userAddressesGetters.isDefault(address);
       if (!isDefault) {
         await remove({ address });
-        await load({});
+        fetch();
```

Comment:
```suggestion
        await fetch();
```

Is dropping the await intentional here? Could be problematic if someone added something new behind the fetch later

Reply:
fetch retriggers useFetch and it does not return a promise so it cannot be awaited.